README.md

```diff
@@ -1,299 +1,83 @@
-# In-toto Attestations
+# in-toto Attestation Framework
 
-This repository defines the in-toto **attestation** format, which represents
-authenticated metadata about a set of software artifacts. Attestations are
-intended for consumption by automated policy engines, such as [in-toto] and
-[Binary Authorization].
+<!-- msm: This is real clunky -->
+The in-toto Attestation Framework is a framework for generating verifiable
```

Comment:
The in-toto Attestation Framework describes mechanisms for generating verifiable claims about how a piece of software is produced. Any developer or consumer of some software can validate its origins and establish trust in its software supply chain.


Not sure this is a whole lot better but:
1. I didn't want to use framework twice right at the start
2. I think splitting it up into two sentences makes it easier to parse

Reply:
Also, best argument to not have "framework" in the name? :smile:

Reply:
Yeah... this is about the only time when having "framework" in the name is inconvenient... But agreed on breaking up that sentence!

Reply:
Please let me know if you like the updated wording better!

Reply:
Loads better!